packages/expo-local-authentication/android/build.gradle

```diff
@@ -84,7 +84,7 @@ android {
 dependencies {
   implementation project(':expo-modules-core')
 
-  api "androidx.biometric:biometric:1.1.0"
+  api "androidx.biometric:biometric:1.2.0-alpha04"
```

Comment:
Generally I'm a bit reluctant to depend on prerelease versions, but looks like this version exists since November and I guess it's not going to be published as stable before we release next SDK so I'm ok with this